entity-service/src/test/kotlin/com/egm/stellio/entity/repository/StandaloneNeo4jSearchRepositoryTests.kt

```diff
@@ -460,38 +518,33 @@ class StandaloneNeo4jSearchRepositoryTests : WithNeo4jContainer {
         )
         createRelationship(EntitySubjectNode(entity.id), "observedBy", partialTargetEntityUri)
 
+        val queryParams = QueryParams(
```

Comment:
this is not the same `queryParams` that was used in the calls to `getEntities` below, you are breaking things here.